### DISCUSSION TITLE
ProVotes

### DISCUSSION THREAD

**Discussion by VimaMT** [Zrips/CMI#6534] [Q&A]
I just bought ProVotes. But like an idiot I did not ask a necessary question first.  :-)

CMI keeps track of votes. It works well with VotingPlugin. Will it also keep track of votes that have been made through ProVotes?

I could set both these plugins up on a test server .. but I really do not want to try voting for a test server to try it out.

**Comment by VimaMT** [comment]
My question may be moot. I added Provotes to my prod server. I received an exception when I tried voting. I'm going to stick with reliable VotingPlugin.

**Comment by TomLewis** [comment]
How did you make it work with VotingPlugin? I have VotingPlugin and CMI is not recording votes from it.

**Comment by VimaMT** [comment]
I simply leave vote counting turned on within CMI's config, but I do not award any prizes through CMI.  This way, VotingPlugin and CMI both see the votes, but only VotingPlugin will reward the player. I can still use the votes in my rankup system within CMI.
